src/app/public/modules/tabs/tabset.service.ts

```diff
@@ -119,13 +132,10 @@ export class SkyTabsetService {
   /**
    * Activates the next registered tab, or the previous one based on a provided array index.
    */
-  private activateNearestTab(arrayIndex: number): void {
-    const newActiveTab = this.tabs[arrayIndex + 1] || this.tabs[arrayIndex - 1];
+  public activateNearestTab(arrayIndex: number): void {
+    const newActiveTab = this.tabs[arrayIndex] || this.tabs[arrayIndex - 1];
```

Comment:
`arrayIndex` is always the actual index, right? Not sometimes an ID like `tabIndex`?